### DISCUSSION TITLE
2024/03/tips-for-getting-un-stuck-on-a-blogging-journey/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#889] [Specific Post Comments]
# 2024/03/tips-for-getting-un-stuck-on-a-blogging-journey/

…if I may be so bold as to offer those.

https://seankilleen.com/2024/03/tips-for-getting-un-stuck-on-a-blogging-journey/

**Comment by mjeaton** [comment]
Great advice!

I would add - Make sure you have a good "why" for your writing. 

Also, writing is a muscle you have to exercise. I would tell people to start writing anything, even a page of "all work and no play makes Jack a dull boy" to help get things flowing. Write every day, even if it's only for 10 minutes. When I get in a rut, I have to get away from my desk because there are two many distractions, so if someone is having a tough time getting started, find a new location. I also keep a writing notebook where I can keep random ideas and where I can flesh out ideas without having to be at my laptop.
